core-wih/src/main/java/io/elimu/a2d2/corewih/ServiceRestAPIDelegate.java

```diff
@@ -79,6 +80,13 @@ public void executeWorkItem(WorkItem workItem, WorkItemManager manager) {
 			ClientRequest clientRequest = (ClientRequest) workItem.getParameter(CLIENT_REQUEST);
 			HttpMethod method = getMethod(workItem);
 
+			int retries = Integer.valueOf(System.getProperty("http.invoke.retries", "3"));
+			int delay = Integer.valueOf(System.getProperty("http.invoke.retrydelay", "5000"));
```

Comment:
I think that this retry delay default is too long. If it fails, the first retry should be in 1 second. Ideally, the delay should implement an incremental backoff (i.e. increase the delay for subsequent retries).

Reply:
Updated the default delay to 1 second. Second retry will be three times the configurable wait delay, and third (and possibly onward retries) will be 5 times configurable delay without increasing for further cases